app/notification.go

```diff
@@ -306,6 +306,18 @@ func (a *App) SendNotifications(post *model.Post, team *model.Team, channel *mod
 					(channelNotification || hereNotification || allNotification),
 					replyToThreadType,
 				)
+			} else {
+				// register that a notification was not sent
+				notificationRegistry := model.NotificationRegistry{
```

Comment:
Why do we want to track these? It just means that the user was online or something that prevented the notification

Reply:
Yes we want to track this when the user is online or the preferences of the user make it so the user don't receive the notification, this is in order to have a way to identify why the user did not receive the notification.